### PR TITLE
Initial stab at a dynamic yaml config doc page.

### PR DESCRIPTION
**Pull request by petey**
This is a pretty hacky way to implement http://browsenpm.org/package.json for our yaml config, for issue #11. 

screen shot 2016-08-02 at 5 07 43 pm

### CODE REVIEW

**Comment by tkyi** [comment]
Super cool, @petey !

File: docs/getting-started/yaml.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <span class="key">steps</span>:
+        <span class="key">init</span>: <span class="value">npm install</span>
+        <span class="key">publish</span>: <span class="value">npm deploy</span></a>
+    <a href="#jobs">...</a>
+</pre>
+    <div class="yaml-side">
+        <div id="workflow" class="hidden">
+            <h4>Workflow</h4>
+            <p>Defines the order of jobs that are executed for the project. All jobs referenced by the workflow must be defined in the jobs section.</p>
+            <p>Jobs can execute in parallel or in series, and in combinations of the two, per this example. Special keywords <strong><em>parallel</em></strong> and <strong><em>series</em></strong> define the flow of the jobs. By default, the jobs in the workflow list are run in series after <em>main</em> job has completed successfully.</p>
```
Comment (tkyi) [inline docs/getting-started/yaml.md]: `Jobs can execute in parallel, in series, or in any combination of the two, per this example.`

File: docs/getting-started/yaml.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        </div>
+        <div id="shared" class="hidden">
+            <h4>Shared</h4>
+            <p>Defines a global configuration that applies to all jobs. Shared configurations are merged with each job, but may be overridden by more specific configuration in a specific job.</p>
+        </div>
+        <div id="environment" class="hidden">
+            <h4>Environment</h4>
+            <p>A set of key/ value pairs for environment variables that need to be set. Any configuration that is valid for a job configuration is valid in shared, but will be overridden by specific job configurations.</p>
```
Comment (tkyi) [inline docs/getting-started/yaml.md]: No space between `key/` and `value`

**Comment by tkyi** [comment]
Should `template replacement` be defined somewhere?
Otherwise, looks like a good first initial stab!

**Comment by petey** [comment]
I added a little more verbiage in image and matrix about templates.

**Comment by tkyi** [comment]
👍 LGTM
giphy-2

**Comment by stjohnjohnson** [comment]
👍 
yess

**Pull request merged**